### DISCUSSION TITLE
How to generate unique JSON file names when using @badeball/cypress-cucumber-preprocessor

### DISCUSSION THREAD

**Discussion by renatasantos** [badeball/cypress-cucumber-preprocessor#1248] [Q&A]
I am using the `@badeball/cypress-cucumber-preprocessor `in my Cypress project and want to integrate the `multiple-cucumber-html-reporter` to generate reports. The` multiple-cucumber-html-reporter` documentation mentions that each JSON file needs to have a unique name to avoid being overwritten.

Could you please guide me on how to configure the Cucumber Preprocessor to ensure that the generated JSON files have unique names?

Thank you in advance for your help!

**Comment by badeball** [comment]
This library generates one, and only one, report per run and this is what you feed into the above-mentioned library. I fail to see how you will need anything more. However, you can change output in a number of ways:

https://github.com/badeball/cypress-cucumber-preprocessor/issues/1055#issuecomment-1631337744

.. or

```
$ cypress run -e jsonOutput=foobar.json
```
